Cargo.toml

```diff
@@ -16,4 +16,4 @@ travis-ci = { repository = "SergioBenitez/yansi", branch = "master" }
 [dependencies]
 
 [dev-dependencies]
-parking_lot = { version = "0.5", features = ["nightly"] }
+serial_test = "*"
```

Comment:
We should never use `*` dependencies, even if they're dev-dependencies, since Cargo can update us to a breaking version without any input of our own.